INSTALLATION.md

```diff
@@ -2,17 +2,36 @@
 
 These instructions should help you get the Cilium Service Mesh-specific images installed in your cluster, with the service mesh features enabled. 
 
-## Install Cilium and Hubble
+## Install Cilium
 
 [Install the Cilium CLI](https://docs.cilium.io/en/v1.11/gettingstarted/k8s-install-default/#install-the-cilium-cli) and run `cilium version` to check that you have version 0.10.2 or above.
 
-Install Cilium with the service mesh builds, and enable Hubble:
+Install Cilium with the service mesh builds. 
+
+* If you want Cilium to co-exist alongside kube-proxy:
+
+```
+cilium install --version -service-mesh:v1.11.0-beta.1 --config enable-envoy-config=true --kube-proxy-replacement=partial --config enable-node-port=true
+```
+
+* If you want to Cilium to completely replace kube-proxy, you will also need to specify the IP address of the node in which the Kubernetes API Server is running: 
+
+```
+cilium install --version -service-mesh:v1.11.0-beta.1 --config enable-envoy-config=true --kube-proxy-replacement=strict --config k8s-api-server=<API server URL>
+```
+
+Note that Cilium Service Mesh will not work with `--kube-proxy-replacement=disabled`. 
```

Comment:
The underlying CiliumEnvoyConfig CRD works for East/West services also without kube proxy replacement or node port. Cilium Ingress requires BPF node port support, though, so Linux kernel must be 4.16 or newer.